content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -69,12 +72,9 @@ security protection. The root private key should be exposed to as few people and
     * `root-cert.pem`: the root certificate
 
     {{< tip >}}
-    You can replace `cluster1` with a string of your choosing. For example, `make mycluster-certs` will
-    result in the creation of a directory called `mycluster`.
-    {{< /tip >}}
-
-    {{< tip >}}
-    To configure additional Istio CAs, you can repeat this step with different cluster/directory names.
+    To configure additional Istio CAs, you can repeat this step with different cluster names.
```

Comment:
Should we say something about why they would do this, i.e. to setup additional CAs in other clusters? It is implied based on the variable names but we should be explicit here, and state that each cluster that has a control plane should have its own intermediate CA. We could say this at the top of the doc and then refer to it here.